frameworks/C++/ULib/benchmark_config.json

```diff
@@ -20,6 +19,24 @@
       "notes": "",
       "versus": ""
     },
+    "json": {
```

Comment:
Why add an entire new test for `json` when it was already included in the default?